mlir/lib/Dialect/SCF/Utils/Utils.cpp

```diff
@@ -970,3 +970,69 @@ scf::ForallOp mlir::fuseIndependentSiblingForallLoops(scf::ForallOp target,
 
   return fusedLoop;
 }
+
+scf::ForOp mlir::fuseIndependentSiblingForLoops(scf::ForOp target,
+                                                scf::ForOp source,
+                                                RewriterBase &rewriter) {
+  // Create fused init_args.
+  auto targetInitArgs = target.getInitArgs();
```

Comment:
Nit: please expand `auto` unless the type is obvious from RHS, e.g., there is a cast.

Reply:
Done throughout.